plugins/pipelines/app/views/samson_pipelines/_deploy_header.html.erb

```diff
@@ -7,10 +7,12 @@
     Triggered deploys:
     <ul>
       <% triggered_deploys.each do |deploy| %>
-        <%= link_to [@project, deploy] do %>
-          <%= deploy.stage&.name %>
-          <%= status_badge deploy.status %>
-        <% end %>
+        <li>
+          <%= link_to [@project, deploy] do %>
+            <%= deploy.stage&.name %>
+            <%= status_badge deploy.status %>
```

Comment:
I kinda would prefer the badge to be outside of the link but either way works...